test/modules/aws-elb-integration.ts

```diff
@@ -221,15 +221,16 @@ describe('ELB Integration Testing', () => {
     WHERE load_balancer_name = '${lbName}';
   `, (res: any[]) => expect(res.length).toBe(0)));
 
-  it('deletes the security group 1', query(`
+  it('deletes the security groups', query(`
     DELETE FROM security_group
-    WHERE group_name = '${sg1}';
+    WHERE group_name = ANY(array['${sg1}', '${sg2}']);
```

Comment:
You shouldn't have to do all of this casting with Postgres arrays, this should work:

```suggestion
    WHERE group_name IN ('${sg1}', '${sg2}');
```

Reply:
Oh I wasn't sure that worked on Postgres